lib/api/utilities/grape_helper.rb

```diff
@@ -38,6 +38,10 @@ def initialize(env)
           @env = env
           @options = {}
         end
+
+        def error!(message, status = nil, headers = nil, backtrace = nil, original_exception = nil)
+          super
+        end
```

Comment:
Not really understanding what this method does here? If it is just calling super with all the arguments passed as well, why does it exist?

Reply:
It is just there to make it public. Could also have used `send` but wasn't sure if that would catch all the cases.